pkg/templates/notificationtemplates/notification_binassets.go

```diff
@@ -0,0 +1,25 @@
+package notificationtemplates
```

Comment:
We could probably move this file in `pkg/templates/assets` to avoid the code duplication.

Reply:
fixed here: e936505542d70ecc34defdf3f036aec30798d88d, i also added comments to WithAssets and WithEmptyCache here: 5dadc99ff8ad9fab80cdf8b1a213dac5e725f533